docs/sources/shared/configuration.md

```diff
@@ -3941,6 +3941,12 @@ otlp_config:
 # CLI flag: -limits.block-ingestion-status-code
 [block_ingestion_status_code: <int> | default = 260]
 
+# List of labels that must be present in the stream. If any of the labels are
```

Comment:
Is this enabled by default?  What happens if no labels are defined?

Reply:
If no labels are defined no labels are enforced, so it just considers the request valid.